airflow/www/static/js/tree/utils/useFilters.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { useSearchParams } from 'react-router-dom';
+
+// Params names
+const BASE_DATE = 'baseDate';
+const NUM_RUNS = 'numRuns';
+const RUN_TYPE = 'runType';
+const RUN_STATE = 'runState';
+const TASK_STATE = 'taskState';
```

Comment:
We may want to pascal_case these strings to keep it consistent with other URL params.

Reply:
You are right my bad -> fixed. And also exported these param name to reuse them in the useGridData hook.